src/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/Helpers.cs

```diff
@@ -118,6 +118,32 @@ public static byte[] FixupKeyParity(this byte[] key)
             }
             return oddParityKey;
         }
+
+        public static void ConvertIntToByteArray(uint dwInput, ref byte[] counter)
```

Comment:
We also typically don't use Hungarian notation.  The parameter should probably be named something more like value and destination.